gui/package.json

```diff
@@ -1,30 +1,41 @@
 {
-    "name": "taipy-gui-core",
-    "version": "2.3.0",
-    "private": true,
-    "devDependencies": {
-        "@types/react": "^18.0.15",
-        "@typescript-eslint/eslint-plugin": "^5.30.7",
-        "@typescript-eslint/parser": "^5.30.7",
-        "child_process": "^1.0.2",
-        "copy-webpack-plugin": "^11.0.0",
-        "dotenv": "^16.0.3",
-        "eslint": "^8.20.0",
-        "eslint-plugin-react": "^7.30.1",
-        "eslint-plugin-react-hooks": "^4.6.0",
-        "eslint-webpack-plugin": "^4.0.0",
-        "ts-loader": "^9.3.1",
-        "typescript": "^5.0.2",
-        "webpack": "^5.74.0",
-        "webpack-cli": "^5.0.1"
-    },
-    "dependencies": {
-        "react": "^18.2.0",
-        "taipy-gui": "file:../../../.virtualenvs/taipy-ow_unobx/lib/site-packages/taipy/gui/webapp"
-    },
-    "scripts": {
-        "install": "node scripts/install.js",
-        "build:dev": "webpack --mode development",
-        "build": "webpack --mode production"
-    }
+  "name": "taipy-gui-core",
+  "version": "2.3.0",
+  "private": true,
+  "devDependencies": {
+    "@mui/icons-material": "^5.0.5",
+    "@mui/lab": "^5.0.0-alpha.53",
+    "@mui/material": "^5.0.6",
+    "@mui/x-date-pickers": "^6.0.0",
```

Comment:
should this be in dependencies ?

Reply:
of course but in dependencies (not devDependencies)

Reply:
Are you referring to only date picker or all @mui packages? @FredLL-Avaiga